3.7/release-notes-new-features37.md

```diff
@@ -8,3 +8,11 @@ Features and Improvements in ArangoDB 3.7
 The following list shows in detail which features have been added or improved in
 ArangoDB 3.7. ArangoDB 3.7 also contains several bug fixes that are not listed
 here.
+
+### Insert-Update
+
+ArangoDB 3.7 adds an insert-update operation that is similar to the already
```

Comment:
to the JS and HTTP API? So
`POST /_api/document/{collection}` and
`db.<collection>.save({ ... }, ... , { overwriteMode: "update", ... })`
are the only affected interfaces?